openedx/features/classrooms/models.py

```diff
@@ -0,0 +1,13 @@
+from nodebb.models import DiscussionCommunity
+from model_utils.models import TimeStampedModel
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class DiscussionCommunityMembership(TimeStampedModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    community = models.ForeignKey(DiscussionCommunity, on_delete=models.CASCADE)
```

Comment:
Kindly test it locally once that it is working, and there are no such constraints on `DiscussionCommunity` table that'd prevent us from adding multiple communities against the same course.

Reply:
Abdullah bhai, I have tested, there are no such constraints.

Reply:
Great, thanks!